.formatter.exs

```diff
@@ -1,12 +1 @@
-[
-  inputs: [
-    "{mix,.formatter}.exs",
-    "{config,lib,test}/**/*.{ex,exs}",
-    "examples/*/{config,lib,priv}/*.ex"
-  ],
-  import_deps: [:protobuf],
-  locals_without_parens: [rpc: 3, intercept: 1, intercept: 2, run: 1, run: 2],
-  export: [
-    locals_without_parens: [rpc: 3, intercept: 1, intercept: 2, run: 1, run: 2]
-  ]
-]
+[subdirectories: ["grpc_core", "grpc_server", "grpc_client"]]
```

Comment:
This file shouldn't exist. top level of the repo should just contain the readme and the folders for each sub-library

Reply:
correct